packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -323,24 +323,20 @@ export class FeatureDevController {
 
         getLogger().info(`${featureName} conversation id: ${session.conversationId}`)
 
+        // This is a loading animation
         this.messenger.sendAnswer({
-            type: 'answer',
+            type: 'answer-stream',
```

Comment:
By this, we're also getting rid of unnecessary additional card just for animation. Awesome!